config/application.yml.default

```diff
@@ -337,6 +337,7 @@ voice_otp_pause_time: '0.5s'
 voice_otp_speech_rate: 'slow'
 voip_block: true
 voip_allowed_phones: '[]'
+in_person_capture_secondary_id_enabled: false
```

Comment:
> LGTM! We have other feature flags using `ipp`. Is that idea to move away from that naming convention?

Yes, the idea is to use `in_person` instead of `ipp`.  This naming convention transcends feature flags.  For example, if you look at git history, you'll see that directories used to say "ipp" instead of "in_person."  The idea is that the full phrase is clearer than an acronym.